### PR TITLE
Fixed os.isdir to work with Windows symbolic links.

### PR DESCRIPTION
**Pull request by tritao**

### CODE REVIEW

File: src/host/os_isdir.c [hunk 1]
```diff
@@ -8,20 +8,33 @@
...
 	/* empty path is equivalent to ".", must be true */
 	if (strlen(path) == 0)
 	{
 		lua_pushboolean(L, 1);
 	}
+#ifdef _WIN32
+	// Use Windows-specific GetFileAttributes since it deals with symbolic links.
+	else if ((attr = GetFileAttributes(path)) != INVALID_FILE_ATTRIBUTES)
+	{
+		int isdir = (attr & FILE_ATTRIBUTE_DIRECTORY) != 0;
```
Comment (samsinsane) [inline src/host/os_isdir.c]: This variable is unused in this scope.

File: src/host/os_isdir.c [hunk 2]
```diff
@@ -8,20 +8,33 @@
...
 
+#ifdef _WIN32
+#include <Windows.h>
+#endif
 
 int os_isdir(lua_State* L)
 {
 	struct stat buf;
 	const char* path = luaL_checkstring(L, 1);
+	int attr;
```
Comment (samsinsane) [inline src/host/os_isdir.c]: This variable causes warnings on non-Windows builds.

**Comment by tritao** [comment]
Thanks for the review, fixed.

**Comment by samsinsane** [comment]
:+1: LGTM! Thanks!

**Pull request merged**